Update release notes for 11.9 (#291)
FairwindsOps/insights-docs: docs/release-notes.md

---
# Release Notes

## 11.9.0
### Bug Fixes and Improvements

> * Fixed the error when visiting `Compliance` page that has no reports

Reviewer: ```suggestion
* Fixed an error when visiting `Compliance` page when there are no reports
```
